{{cookiecutter.project_slug}}/Pipfile

```diff
@@ -41,8 +41,8 @@ python_version = "3.9"
 [dev-packages]
 # Code quality
 # ------------------------------------------------------------------------------
-flake8 = "==3.9.2"  # https://github.com/PyCQA/flake8
-flake8-isort = "==4.0.0"  # https://github.com/gforcada/flake8-isort
+flake8 = "==4.0.1"  # https://github.com/PyCQA/flake8
+flake8-isort = "==4.1.1"  # https://github.com/gforcada/flake8-isort
```

Comment:
I don't think we should be version pinning by default in the Pipfile. That's what the lock file is for. Perhaps just change these to `"*"`?

Reply:
I think as a general rule it is best to at least specify major versions, and maybe even minor versions (so either `4.*` or `4.0.*` in this case). 

For well-supported and slow-moving packages like flake8, I am more inclined towards `"*"`, because I think the risk is low...

But I want to be clear that I would not do that for just any package - and even in this case there is some risk of an uncontrolled break. Not every package actually uses semver, and not all packages do semver correctly, even if they try. I have seen patch versions break builds more than once. That's even with the lock file, because in practice devs don't hesitate to destroy or update the lock. The Pipfile.lock _does_ help us trace breaking changes, but it doesn't necessarily protect us from a package being updated automatically - the Pipfile can do that, though.

Reply:
Yea we talked about this before and I don't want to take this PR back down that rabbit hole. 
(I have mixed opinions in my head about your argument here) 
So we can move on for now.

The one thought I'll leave here is that when a team starts a project from scratch, they usually install the latest of everything to start and then peg everything. Then the risk becomes no one ever taking the time to version bump. This risk is amplified if teams are bootstrapping new projects with a Pipfile we've provided that suffers from this same outdated issue. So you are starting a new project with an already outdated Pipfile.

Something to stew on. We can discuss later if there is a non-process way to mitigate the outdated Pipfile issue.